scrunch/datasets.py

```diff
@@ -319,14 +420,30 @@ def __repr__(self):
     def __str__(self):
         return self.name
 
+    @property
+    def members(self):
+        return Members(self.resource)
+
+    def get_dataset(self, dataset):
+        try:
+            shoji_ds = self.resource.datasets.by('name')[dataset].entity
+        except KeyError:
+            try:
+                shoji_ds = self.resource.datasets.by('id')[dataset].entity
+            except KeyError:
+                raise KeyError(
+                    "Dataset (name or id: %s) not found in project." % dataset)
+        ds = BaseDataset(shoji_ds)
+        return ds
+
     @property
     def users(self):
         """
-        :return: dictionary of User instances
+        TODO: deprecate in favor of members.list property
         """
-        # TODO: return a dictionary keyed by email and values should be User
-        # instances, but when trying got 403 from Crunch and we need
-        # support for teams now
+        LOG.warning("""This method is legacy and will be deprecated
+            in future releases. Please make use of project.members.list()
+            instead""")  # noqa: E501
```

Comment:
Why not rewrite the body of these functions to just use the new code, then they're only shortcuts and not duplicate of logic.  Same for the other deprecated methods.

Reply:
good idea.